packages/zod-nestjs/src/lib/create-zod-dto.ts

```diff
@@ -1,6 +1,8 @@
 import type { SchemaObject } from 'openapi3-ts';
 import { generateSchema, OpenApiZodAny } from '@anatine/zod-openapi';
 import * as z from 'zod';
+import type { TupleToUnion, Merge } from 'type-fest';
```

Comment:
Never ideal to introduce a new dependency, but the nice thing is it is purely types, no JS.
We could try to extract the logic if need be.